### PR TITLE
Add the right-click tip to dis/enable Data-defined override button

### PR DESCRIPTION
**Pull request by DelazJ**

### CODE REVIEW

File: source/docs/user_manual/introduction/general_tools.rst [hunk 1]
```diff
@@ -1188,27 +1188,34 @@ this tool allows you to set dynamic value for the concerned parameter. When enab
...
-   the icon is red |dataDefineError| or |dataDefineExpressionError|.
-
-
 Parameters that can be used with data-defined tools are:
 
 * Style and symbols parameters
 * Labels parameters
 * Composer parameters
 
+.. tip:: **Use right-click to (de)activate the data overriding**
```
Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: I think one line is enough for this information. But we already discussed about this in the past. As an user, I appreciate that documentation goes straight to the point. Readers want to learn fast and too much text means reading over the text.

Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: > Readers want to learn fast and too much text means reading over the text.

Sure (and this is something I need to improve myself on and somehow, this is some of the reasons reviews are needed)! But note that, IF the DD button hasn't been set beforehand, right-clicking doesn't have any effect.
That said, if you have a better wording, please propose and I improve this...

Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: @yjacolin how about?
> you can enable or disable a configured |dataDefined| data-defined override button by simply clicking the widget with the mouse right button

Short enough?

Comment (yjacolin) [inline source/docs/user_manual/introduction/general_tools.rst]: perfect!

Comment (DelazJ) [inline source/docs/user_manual/introduction/general_tools.rst]: merci!

**Pull request merged**